webapp/package.json

```diff
@@ -9,32 +9,53 @@
         "fix": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx . --quiet --fix"
     },
     "dependencies": {
-        "core-js": "3.1.4",
-        "mattermost-redux": "github:mattermost/mattermost-redux#b56e7e3a6f7f5de281fd3541eefec9345a8bbc89",
+        "@primer/octicons-react": "10.1.0",
+        "core-js": "3.6.5",
+        "csstype": "3.0.3",
+        "mattermost-redux": "5.27.0",
```

Comment:
I noticed there was a very specific version of redux. Was it for a specific reason? my question is like, is there any risk of regression for upgrading to 5.27?

Reply:
There is a type defined in mattermost-redux named "Theme" it was not present in the version specified earliar, that's why updated it to this version.